charts/rancher-thanos/v0.0.1/values.yaml

```diff
@@ -0,0 +1,156 @@
+# Default values for thanos.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+nameOverride: ""
+fullnameOverride: ""
+
+# column split Ids, e.g., c-abc:c-def
+clusterIds: ""
+
```

Comment:
`clusterIds` is configurable? Our controller will override it. So it's not something end users can configure.  Maybe we should remove it or add a comment says read only field? 
Same for other readonly fields.